website/prereg/utils.py

```diff
@@ -1,9 +1,33 @@
 from modularodm import Q
 
-def get_prereg_schema():
+from website.util import permissions
+
+
+PREREG_CAMPAIGNS = {
+    'prereg': 'Prereg Challenge',
+    'erpc': 'Election Research Preacceptance Challenge',
+}
+
+def drafts_for_user(user, campaign):
```

Comment:
It has at least one test: `test_drafts_for_user_omits_registered` in `tests/test_prereg.py`